### PR TITLE
feat(docs): Add Traefik logs doc

### PR DESCRIPTION
**Pull request by crazyuploader**
## Pull Request

---

### 📄 Summary
This PR adds doc for Traefik

Doc: [Link](https://staging.signoz.io/docs/integrations/opentelemetry-traefik/)

#### Issues closed by this PR
Closes https://github.com/SigNoz/growth-pod/issues/1062

---

### ✅ Change Type
_Select all that apply_

- [ ] ✨ Feature
- [ ] 🐛 Bug fix
- [ ] ♻️ Refactor
- [ ] 🛠️ Infra / Tooling
- [ ] 🧪 Test-only

---

---

- Tests added/updated:
- Manual verification:
- Edge cases covered:

---

- Blast radius:
- Potential regressions:
- Rollback plan:

---

| Field | Value |
|------|-------|
| Deployment Type | Cloud / OSS / Enterprise |
| Change Type | Feature / Bug Fix / Maintenance |
| Description | User-facing summary |

---

### 📋 Checklist
- [ ] Tests added or explicitly not required
- [ ] Manually tested
- [ ] Breaking changes documented
- [ ] Backward compatibility considered

---

## 👀 Notes for Reviewers

---

### CODE REVIEW

File: data/docs/integrations/opentelemetry-traefik.mdx [hunk 1]
```diff
@@ -0,0 +1,389 @@
...
+
+## Collect Traefik access logs
+
+Traefik writes the access log itself. The Collector tails that file, parses each JSON line into fields, and forwards the result to SigNoz.
+
+### Step 1: Enable JSON access logs
+
+Traefik ships with access logs off, and its `common` default format needs regex parsing. Turn access logs on and switch the format to `json`.
+
+Using CLI flags:
```
Comment (revmag) [inline data/docs/integrations/opentelemetry-traefik.mdx]: Put this in tabs

File: data/docs/integrations/opentelemetry-traefik.mdx [hunk 2]
```diff
@@ -0,0 +1,389 @@
...
+description: Ship Traefik access logs to SigNoz with the OpenTelemetry Collector. Parse the JSON format into structured fields to query routers, services, and status codes.
+doc_type: howto
+---
+
+## Overview
+
+Traefik writes two kinds of logs:
+
+- **Access logs:** one record per HTTP request, with the router, backend service, status code, and timings. This guide collects these.
+- **Application logs:** Traefik's own startup and configuration events, controlled by the separate `log` option.
```
Comment (revmag) [inline data/docs/integrations/opentelemetry-traefik.mdx]: explain it more clearly, whats the separate log option here?

**Review by revmag — APPROVED** [review]
